src/routes/pricing/success/+page.svelte

```
@@ -0,0 +1,120 @@
+						<span>Advanced cohort analytics</span>
+					</li>
+					<li class="flex items-start gap-3">
+						<Sparkles size={16} class="text-success mt-0.5" />
+						<span>Priority processing for all tasks</span>
+					</li>
+				</ul>
+			</div>
+
+			<!-- CTA -->
+			<div class="flex flex-col sm:flex-row gap-3 justify-center">
+				<a href="/admin/question-studio" class="btn btn-primary">
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)